suite-native/module-settings/src/slice.ts

```diff
@@ -1,19 +1,21 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 
-import { ThemeColorVariant } from '@trezor/theme';
+import { fiatCurrencies, Currency, CurrencyType } from '@suite-common/suite-config';
 
-export type AppColorScheme = ThemeColorVariant | 'system';
+import { AppColorScheme } from './types';
 
 export interface AppSettingsState {
     colorScheme: AppColorScheme;
+    fiatCurrency: Currency;
 }
 
-type SliceState = {
+type SliceRootState = {
```

Comment:
It would be much clearer if this type, as well as the file name actually reflected what slice it is. `SettingsSliceRootState`, `settingsSlice.ts` 

Also, why bother with those root types in slices? @Nodonisko  What if a selector is more complex and needs access to other slices? Why not have the actual `RootState` in there? 🤔  We are still working with one state in here.

Even if you think in the context of sharing slices between Native and Web, it still makes little sense to me to see the `RootState` separately in each slice.

Reply:
Yeah, it's probably better name. If selector is more complex and needs access to other slices, we should always use `createSelector` function, where you can just import selector without the need to import types so it shouldn't really matter imo. 

With rootState, do you mean the combined reducers state? This shouldn't be imported from the root state package because it would create circular dependency and this type is enough to type this slice / portion of the state.

Reply:
I will hopefully have ready explanation about RootState and selectors in code style guide soon. About the file names, I am working on that too, what is @dahaca suggesting is one of possible options, currently most favored by me, but give me time to evaluate it properly.

I hope I will have this finished by start of next week, so we will refactor it then. I don't want to block this PR by discussion about conventions that we don't even have defined yet.

Reply:
Yep, will use SettingsSliceRootState :)

Reply:
https://github.com/trezor/trezor-suite/pull/5895/commits/285dce9e0a56ad98fd9ab9555c697cffc3a30539

Reply:
@Nodonisko Hopefully I will have time soon to add my ideas to the code style drafts too :)